src/cleanlab_codex/response_validation.py

```diff
@@ -88,18 +90,75 @@ class BadResponseDetectionConfig(BaseModel):
 
 _DEFAULT_CONFIG = BadResponseDetectionConfig()
 
+# Type aliases for validation scores
+SingleScoreDict = dict[str, float]
+NestedScoreDict = OrderedDict[str, SingleScoreDict]
+
+"""Type alias for validation scores.
+
+Scores can be either a single score or a nested dictionary of scores.
+
+Example:
+    # Single score
+    scores: ValidationScores = {"score": 0.5}
+    # Nested scores
+    scores: ValidationScores = {
+        "check_a": {"sub_score_a1": 0.5, "sub_score_a2": 0.5},
+        "check_b": {"sub_score_b1": 0.5, "sub_score_b2": 0.5},
+    }
+"""
```

Comment:
More of an illustrative example for the reviewer. Not sure how much value this will add to our docs, but could use some guidance on where to document these two types of score dictionaries.